tests/lib/FreeBSDTest.php

```diff
@@ -97,54 +98,46 @@ public function testGetMemoryNoData(): void {
 	}
 
 	public function testGetNetworkInterfaces(): void {
+		$this->os->method('getNetInterfaces')
+			->willReturn(include __DIR__ . '/../data/freebsd_net_get_interfaces.php');
 		$this->os->method('executeCommand')
 			->willReturnCallback(static function ($command) {
-				if ($command === '/sbin/ifconfig -a') {
-					return file_get_contents(__DIR__ . '/../data/freebsd_interfaces');
-				}
-				if ($command === '/sbin/ifconfig lo0') {
-					return file_get_contents(__DIR__ . '/../data/freebsd_interface_lo0');
-				}
 				if ($command === '/sbin/ifconfig pflog0') {
 					return file_get_contents(__DIR__ . '/../data/freebsd_interface_pflog0');
 				}
 				if ($command === '/sbin/ifconfig epair0b') {
 					return file_get_contents(__DIR__ . '/../data/freebsd_interface_epair0b');
 				}
-
-				// Regex matches way more than the interface names, so if it doesn't match any of the defined ones, throw.
-				throw new \RuntimeException();
```

Comment:
Removing this means other commands are executed? Which ones? I’d feel better if the test lists them and throw for the rest.

Reply:
Added the RuntimeException back.